src/shared/state/api-slices/graphqlApiSlice.ts

```diff
@@ -17,50 +17,46 @@
 import { createApi } from '@reduxjs/toolkit/query/react';
 import { request, ClientError } from 'graphql-request';
 
-import config from 'config';
-
-const graphqlBaseQuery =
-  ({ baseUrl }: { baseUrl: string }) =>
-  async ({
-    body,
-    variables = {}
-  }: {
-    body: string;
-    variables?: Record<string, string | number | boolean>;
-  }) => {
-    try {
-      const result = await request(baseUrl, body, variables);
-      return { data: result };
-    } catch (error) {
-      if (error instanceof ClientError) {
-        const { name, message, stack, request, response } = error;
-        return {
-          error: {
-            status: error.response.status,
-            meta: {
-              data: error.response.data ?? null,
-              errors: error.response.errors
-            },
-            name,
-            message,
-            stack
-          },
-          meta: { request, response }
-        };
-      }
+const graphqlBaseQuery = async ({
+  body,
+  url,
+  variables = {}
+}: {
+  url: string;
+  body: string;
+  variables?: Record<string, string | number | boolean>;
+}) => {
+  try {
+    const result = await request(url, body, variables);
+    return { data: result };
+  } catch (error) {
+    if (error instanceof ClientError) {
+      const { name, message, stack, request, response } = error;
       return {
         error: {
-          status: 500,
-          meta: { error: (error as Error).message }
-        }
+          status: error.response.status,
```

Comment:
Just use `response` instead of `error.response` since it is already available?

Reply:
✅